src/pages/Admin/Admin_v2.tsx

```diff
@@ -163,7 +164,7 @@ const Admin_v2: React.FC = () => {
         onHide={() => {}}
```

Comment:
[OOS] It would be good if `onHide` was an optional callback here, rather than needing to pass it a noop function.